packages/panna-sdk/src/react/components/buy/select-buy-provider-step.tsx

```diff
@@ -139,13 +110,6 @@ export function SelectBuyProviderStep({ form }: SelectBuyProviderStepProps) {
           </div>
         ) : (
           <>
-            {createSessionError && (
-              <div className="border-destructive/50 bg-destructive/10 flex items-center justify-center rounded-md border px-4 py-3">
-                <Typography variant="muted">
-                  Failed to create onramp session. Please try again.
-                </Typography>
-              </div>
-            )}
             {availableProviders.map((provider) => (
```

Comment:
Below, we're mapping all the available providers with the same quote fetched from `onramp` instead. This needs fixing!